reference/python-sdk/v3.2.md

```diff
@@ -0,0 +1,1472 @@
+Creates a transition handle, calls the POST /transitions endpoint.
+
+```python
+>>> import json
+>>> from pathlib import Path
+>>> from las.client import Client
+>>> client = Client()
+>>> in_schema = {'$schema': 'https://json-schema.org/draft-04/schema#', 'title': 'in', 'properties': {...} }
+>>> out_schema = {'$schema': 'https://json-schema.org/draft-04/schema#', 'title': 'out', 'properties': {...} }
+>>> # A typical docker transitions
+>>> docker_params = {
+>>>     'imageUrl': '<image_url>',
+>>>     'credentials': {'username': '<username>', 'password': '<password>'}
+>>> }
+>>> client.create_transition('docker', in_schema=in_schema, out_schema=out_schema, params=docker_params)
+>>> # A typical manual transitions
+>>> assets = {'jsRemoteComponent': 'las:asset:<hex-uuid>', '<other asset name>': 'las:asset:<hex-uuid>'}
+>>> manual_params = {'assets': assets}
+>>> client.create_transition('manual', in_schema=in_schema, out_schema=out_schema, params=manual_params)
+```
+
+
+* **Parameters**
+
+    
+    * **in_schema** (*dict*) – Json-schema that defines the input to the transition
+
+
+    * **out_schema** (*dict*) – Json-schema that defines the output of the transition
+
+
+    * **name** (*str*) – Name of the transition
+
+
+    * **transition_type** (*str*) – Type of transition “docker”|”manual”
+
+
+    * **params** (*Optional**[**dict**]*) – Parameters to the corresponding transition type
+
+
+    * **description** (*Optional**[**str**]*) – Description of the transition
+
+
+
+* **Returns**
+
+    Transition response from REST API
+
+
+
+* **Return type**
+
+    dict
+
+
+
+* **Raises**
+
+    `InvalidCredentialsException`, `TooManyRequestsException`, `LimitExceededException`, `requests.exception.RequestException`
+
+
+
+#### create_user(email: str)
+Creates a new user, calls the POST /users endpoint.
+
+```python
+>>> from las.client import Client
+>>> client = Client()
+>>> client.create_user('<email>')
+```
+
+
+* **Parameters**
+
+    **email** (*str*) – Email to the new user
+
+
+
+* **Returns**
+
+    User response from REST API
+
+
+
+* **Return type**
+
+    dict
+
+
+
+* **Raises**
+
+    `InvalidCredentialsException`, `TooManyRequestsException`, `LimitExceededException`, `requests.exception.RequestException`
+
+
+
+#### create_workflow(specification: dict, \*, error_config: Optional[dict] = None, \*\*optional_args)
+Creates a new workflow, calls the POST /workflows endpoint.
+
+```python
+>>> from las.client import Client
+>>> from pathlib import Path
+>>> client = Client()
+>>> specification = {'language': 'ASL', 'version': '1.0.0', 'definition': {...}}
+>>> error_config = {'email': '<error-recipient>'}
+>>> client.create_workflow(specification, error_config=error_config)
+```
+
+
+* **Parameters**
+
+    
+    * **specification** (*dict*) – Specification of the workflow,
```

Comment:
Is it an idea to quickly explain with features from ASL is supported / not supported and what the 'resource' key should be in our case?  (i.e. a transition id)

Reply:
I think that belongs in a tutorial and not in a method specification don't you agree?

Reply:
Yes agreed